operator/pkg/securedcluster/values/translation/translation.go

```diff
@@ -167,12 +172,18 @@ func (t Translator) checkInitBundleSecret(ctx context.Context, sc platform.Secur
 	return nil
 }
 
-func (t Translator) getSensorValues(sensor *platform.SensorComponentSpec) *translation.ValuesBuilder {
+func (t Translator) getSensorValues(sensor *platform.SensorComponentSpec, config scanner.AutoSenseResult) *translation.ValuesBuilder {
 	sv := translation.NewValuesBuilder()
 
-	sv.AddChild(translation.ResourcesKey, translation.GetResources(sensor.Resources))
-	sv.SetStringMap("nodeSelector", sensor.NodeSelector)
-	sv.AddAllFrom(translation.GetTolerations(translation.TolerationsKey, sensor.Tolerations))
+	if sensor != nil {
+		sv.AddChild(translation.ResourcesKey, translation.GetResources(sensor.Resources))
+		sv.SetStringMap("nodeSelector", sensor.NodeSelector)
+		sv.AddAllFrom(translation.GetTolerations(translation.TolerationsKey, sensor.Tolerations))
+	}
+
+	if config.EnableLocalImageScanning {
+		sv.SetPathValue("localImageScanning.enabled", strconv.FormatBool(config.EnableLocalImageScanning))
```

Comment:
Nit: This shows that maybe it would be better to use `LocalImageScanningEnabled` or even `ImageScanningAvailableLocally` or `ImageScanningAvailableInNamespace`, instead of `EnableLocalImageScanning` as a field for `AutoSenseResult`. Same goes for Helm value key `localImageScanning.enabled`

Reply:
Sensor should be aware whether to enable or disable the scanning feature which was done before by `ROX_USE_LOCAL_SCANNING`.
This setting sets that env variable, it does not determine whether the Scanner is in the same namespace. 
If Scanner is not in the same namespace one could still set a custom endpoint for Scanner on Sensor.